docs/user-guide/installation/f24+.rst

```diff
@@ -0,0 +1,282 @@
+
+::
+
+   $ sudo dnf install mongodb-server
+
+You need mongodb-server with version >= 2.4 installed for Pulp server. It is highly recommended
+that you `configure MongoDB to use SSL`_. If you are using Mongo's authorization feature, you
+will need to grant the ``readWrite`` and ``dbAdmin`` roles to the user you provision for Pulp to
+use. The ``dbAdmin`` role allows Pulp to create collections and install indices on them.
+
+After installing MongoDB, you should configure it to start at boot and start it::
+
+ $ sudo systemctl enable mongod
+ $ sudo systemctl start mongod
+
+.. warning::
+   On new MongoDB installations, MongoDB takes some time to preallocate large files and will not
+   accept connections until it finishes. When this happens, Pulp will wait for MongoDB to
+   become available before starting.
+
+.. _configure MongoDB to use SSL: http://docs.mongodb.org/v2.4/tutorial/configure-ssl/#configure-mongod-and-mongos-for-ssl
+
+
+Message Broker
+--------------
+
+You must also provide a message broker for Pulp to use. Pulp will work with Qpid or RabbitMQ, but
+is tested with Qpid and uses Qpid by default. This can be on the same host that you will
+run Pulp on, or elsewhere as you please.
+
+
+qpidd
+^^^^^
+
+To install qpidd, run this command on the host you wish to be the message broker:
+
+::
+
+   $ sudo dnf install qpid-cpp-server qpid-cpp-server-linearstore
+
+Pulp uses the ``ANONYMOUS`` Qpid authentication mechanism by default. To
+enable username/password-based ``PLAIN`` broker authentication, you will need
+to configure SASL with a username/password, and then configure Pulp to use that
+username/password. Refer to the Qpid docs on how to configure username/password
+authentication using SASL. Once the broker is configured, configure Pulp according
+to the docs on using
+:ref:`Pulp with Qpid and username/password authentication <pulp-broker-qpid-with-username-password>`.
+
+The server can be *optionally* configured so that it will connect to the broker using SSL by
+following the steps defined in the :ref:`Qpid SSL Configuration Guide <qpid-ssl-configuration>`.
+By default, Pulp does not expect to use SSL and will connect to the broker using a plain TCP
+connection to localhost.
+
+After installing and configuring Qpid, you should configure it to start at boot and start it:
+
+::
+
+   $ sudo systemctl enable qpidd
+   $ sudo systemctl start qpidd
+
+
+RabbitMQ
+^^^^^^^^
+
+On the host you wish to use as your message broker, run this command To install RabbitMQ::
+
+   $ sudo dnf install rabbitmq-server
+
+After installing and configuring RabbitMQ, you should configure it to start at boot and start it::
+
+   $ sudo systemctl enable rabbitmq-server
+   $ sudo systemctl start rabbitmq-server
+
+
+Pulp Server
+^^^^^^^^^^^
+
+Now we are ready to install and configure the Pulp server!
+
+#. Install the Pulp server, task workers, and dependencies. You can get the packages you need by
```

Comment:
Should this be broken into two steps with the addition of `sudo dnf install pulp-server` being one of them? Also for it to work with `qpidd` as the broken it will not start without the installation of `python-qpid` and `qpid-tools`. This mimics an aspect of the [`pulp-server-qpid` definition in comps.xml](https://github.com/pulp/pulp/blob/73f178121522563fd9e8809ea824be409825f13a/comps.xml#L31-L32).

The `qpid-tools` is used by `goferd` but also required by `pulp-server`.

Am I thinking about this right?